index.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html>
+
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins&display=swap">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+    integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+    crossorigin="anonymous" referrerpolicy="no-referrer">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+  <title>Mobile Portfolio</title>
+  <header>
+    <img src="img/My Logo.svg" alt="humburger" class="Logo">
+    <img src="img/Union.svg" alt="humburger" class="Union">
+  </header>
+</head><br><br><br><br>
+
+
+<body>
+  <div id="my-section">
+    <section class="container">
+      <h1>I'm Edgah.<br>
+        Glad to see<br>
+        you!</h1>
+      <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <img src="img/LET’S CONNECT.svg" alt="humburger"><br><br>
+    </section>
+
+
+    <section>
+      <ul class="social-icons">
+        <li>
+          <img src="img/twitter icon.svg">
+        </li>
+        <li>
+          <img src="img/Linkedin icon.svg">
+        </li>
+        <li>
+          <img src="img/Mobile icon.svg">
+        </li>
+        <li>
+          <img src="img/git icon.svg">
+        </li>
+        <li>
+          <img src="img/peace icon.svg">
+        </li>
+      </ul>
+    </section><br><br><br><br><br><br><br><br>
+  </div>
+
+
+  <div>
+    <section class="together">
+      <img src="img/Snapshoot Portfolio.svg">
+      <div class="textp">
+        <h2>Tonic</h2>
+        <p class="inline-paragraph" id="canopy">CANOPY</p>
+        <img src="img/dot.svg">
+        <p class="inline-paragraph" id="devend">Back End Dev</p>
+        <img src="img/dot.svg">
+        <p class="inline-paragraph" id="unique">2023</p>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+      <button type="button" class="see-project">See Project</button>
```

Comment:
- [ ] According to your previouse reviewer kindly Please make sure you added the hover and active properties to your button. Kindly check the Figma design,The button (e.g., "Get my resume,See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design.
- [ ] ![Screenshot 2023-06-04 000218](https://github.com/Edgahkipkemoi/mobile/assets/92458236/e909995b-4946-403c-8285-ab721d671e93)
- [ ] see screenrecorf  

https://github.com/Edgahkipkemoi/mobile/assets/92458236/87f3ff1c-087a-4849-b717-a889596b1628